MovingBlocks/Terasology
# Input settings menu i18n; misc. i18n fixes

### Contains

An implementation for #2309. Adds internationalization support to static strings, categories and binding names in the Input Settings menu - if a category/binding name is a URN to an internationalisable string, it will be translated, otherwise it will fall back to the default (presumably English) string value

Other miscellaneous additions:

- Sorted the language files by key (via https://www.npmjs.com/package/sort-json). Doesn't break anything but makes the diff look spookier than it actually is.
- Fixed the mess in the Galician file - it used some empty string values, causing awkward UI issues. Not that this particular translation was ever usable :confused:

### How to test

Go to Settings → Player, switch the language to Ukrainian, return to Settings → Input (the fourth button from the top :wink:) and check out the translated menu!

![Translated menu screenshot](https://cloud.githubusercontent.com/assets/13783592/15049551/82bc0998-12f7-11e6-8d7d-0b11bc262f54.png)

### Outstanding before merging

- [ ] The Inventory category has some strings specific to the Core module. If these are to be translated, they will require a new set of .lang files and a separate Weblate subproject.
